### PR TITLE
Lipsync Update

### PR DESCRIPTION
**Pull request by AirSurfer09**
Patch : 0.0.7
- Added Lipsync to ConvaiClient
- Added Helper Function OvrToMorph for converting viseme to morph targets
- Package Json updated with description and keywords.
- Readme Updated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,11 +9,14 @@ import { ConvaiClient } from 'convai-web-sdk';
...
-    apiKey: <add convai api key>,
-    characterId: <add convai character id>,
-    enableAudio: true // use false for text only.
-});
+const convaiClient = useRef(null);
+convaiClient.current = new ConvaiClient({
+      apiKey: string //Enter your API Key here,
+      characterId: string //Enter your Character ID,
+      enableAudio: boolean, //use false for text only.
+      sessionId: string //use this to save conversation sessions to save conversations
```
Comment (crazylazylife) [inline README.md]: What is this save conversation session for.

File: package.json [hunk 2]
```diff
@@ -1,7 +1,26 @@
 {
   "name": "convai-web-sdk",
-  "version": "0.0.6",
+  "discription":"Integrate lifelike digital beings into your web applications with real-time conversations, actions, and facial expressions. Supports a variety of voices, languages, and emotions.",
+   "keywords": [
```
Comment (crazylazylife) [inline package.json:4]: Is this for SEO?

**Review by crazylazylife — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (AirSurfer09) [reply package.json:4]: Yes

File: package.json [hunk 3]
```diff
@@ -1,7 +1,26 @@
 {
   "name": "convai-web-sdk",
-  "version": "0.0.6",
+  "discription":"Integrate lifelike digital beings into your web applications with real-time conversations, actions, and facial expressions. Supports a variety of voices, languages, and emotions.",
```
Comment (mousoom) [inline package.json]: typo "description"

**Review by mousoom — APPROVED** [review]

File: package.json (hunk 3 above)
Comment (AirSurfer09) [reply package.json]: Done

**Pull request merged**